### ISSUE TITLE
Recent Activity (user submissions) time zone issues

### ISSUE THREAD

**Issue by RyanTG** [scottwainstock/pbm#1143]
I believe:
- We show the last 30 days of user submissions in Recent Activity
- But submissions that you just made are not showing up immediately because... they're in the future?

Solution? I dunno. Offset the 30 days to take into account the difference between now and the server's time?

**Comment by scottwainstock** [comment]
@RyanTG Would this offset be dependent on the local time in the APP though? 

```
scottwainstock@Scotts-MacBook-Pro ~/work/pbm (master) $ heroku config -a glowing-dusk-5085 | grep TZ
TZ:                                    America/Los_Angeles
```

We're in PST... "now" will be dependent on who's looking. You know? Maybe I need to make the endpoint allow for a variable "last X days" thing and then the app decides what the X is?

**Comment by RyanTG** [comment]
Yeah it does seem like the now will be dependent on where the user is. So the app may need to figure that out. 

This is assuming we’ve properly identified the issue here.

**Comment by RyanTG** [comment]
If the server is in PST, and you and I are physically in PST, then why would the recent activity be delayed as it is? 

When it does finally show up in the app, it shows the correct timestamp. For example, the activity will occur on 4/10, but won’t show up until 4/11 in the feed, and the date for it will say 4/10. 

I guess the issue could be similar to the one we saw recently with the operator email digests: the NOW was inputted incorrectly in the code.

**Comment by scottwainstock** [comment]
Maybe this? https://github.com/scottwainstock/pbm/commit/f790acde3ca5a331b31135224de1a9f88a163a18

**Comment by RyanTG** [comment]
YES! 

Thanks.

**Comment by RyanTG** [comment]
Hmm, reopening. This is still an issue on live. I can’t confirm what the _cause_ is - I just know that user submissions aren’t showing up under that endpoint right away. I can do more testing.

**Comment by RyanTG** [comment]
The submissions at the above link are now showing up. Here’s one:

{"id":103019,"submission_type":"new_lmx","submission":"Dr. Dude was added to Barcade LA by pbm","region_id":5,"created_at":"2019-05-07T01:25:38.482Z","updated_at":"2019-05-07T01:25:38.482Z","user_id":13,"location_id":12160,"machine_id":738}

Time stamp is showing 5/7 1:15am. But I actually made this submission at 5/6 6:25pm.

**Comment by RyanTG** [comment]
Thiiiiiiiiiis seems to work! https://github.com/scottwainstock/pbm/pull/1167/commits/ea973fdea499c942cd494992e9d6dcfa332d958d

I tested it by:
Having it commented out completely. Result: the submissions weren't showing up in the end point.
Adding the Pacific default time zone. Result: submissions show up right away.
I also tried adding the `config.active_record.default_timezone` to `local`. Result: submissions show up right away. But since the above test worked, I commented this out for now.
I also used a VPN to pretend I was in a different time zone. And made a submission in that time zone. Result: submissions showed up right away.

So I think this might be good. Next step is to sort them by date.
